### PR TITLE
Feature/add backup methods with crate

### PR DESCRIPTION
**Pull request by rem-code-s**

### CODE REVIEW

File: src/child/src/methods.rs
Comment (Web3NL) [inline src/child/src/methods.rs:1]: Why is this part of this PR?

File: src/child/src/backup.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+use ic_cdk::caller;
+use ic_cdk_macros::{query, update};
+use ic_scalable_canister::store::Data;
+use shared::profile_models::Profile;
+
+#[update(guard = "is_owner")]
+#[candid_method(update)]
+fn backup_data() -> String {
+    DATA.with(|data| {
+        let serialized = Encode!(&*data).expect("Failed to encode data");
```
Comment (Web3NL) [inline src/child/src/backup.rs:13]: again, consider immediately deserializing after this line and discard the result

**Review by Web3NL — APPROVED** [review]

File: src/child/src/methods.rs
Comment (rem-code-s) [reply src/child/src/methods.rs:1]: i have no idea 🤔

Comment (rem-code-s) [reply src/child/src/methods.rs:1]: Oh this is a bugfix

**Pull request merged**